st2common/st2common/models/db/__init__.py

```diff
@@ -59,6 +60,13 @@ def db_teardown():
     mongoengine.connection.disconnect()
 
 
+def convert_to_python(instance):
```

Comment:
Uh sure.  If didn't have docs to begin with.  I just consolidated the few lines of code here.